Source/WebCore/platform/graphics/gstreamer/eme/WebKitCommonEncryptionDecryptorGStreamer.h

```diff
@@ -26,6 +26,7 @@
 
 #include <gst/base/gstbasetransform.h>
 #include <gst/gst.h>
+#include <stdint.h>
```

Comment:
Why are you adding this? I don't think it is necessary considering that we are not touching the file, debugging leftover?